docs/deploy.md

```diff
@@ -1,25 +0,0 @@
-# Deploying
-
-If you would like to run your own instance of this plugin, see the [docs for deploying plugins](https://github.com/probot/probot/blob/master/docs/deployment.md).
```

Comment:
i dont think the plugins approach is even valid anymore, so i captured the idea of this more as just forking the repo.